systers_portal/static/css/style.css

```diff
@@ -247,6 +251,18 @@ table.decoration-none  tr a {
   margin: 30px 0px;
 }
 
+.text-indent35 {
+  text-indent: 35px;
+}
+
+/* Meetup Location Members Grid CSS
+-------------------------------------------------- */
+.crop_img {
+  width: 100px;
+  height: 100px;
+  overflow: hidden;
+}
+
```

Comment:
After making modifications as suggested above http://dpaste.com/0HBR9YZ grid looks like http://gyazo.com/6dbb128e8d3992264e6cdba744f344f7 . Images are no more of coming in same size.